src/test/kotlin/lotto/domain/LottoStatisticsTest.kt

```diff
@@ -0,0 +1,63 @@
+package lotto.domain
+
+import lotto.view.LottoOutputView
+import org.assertj.core.api.Assertions.assertThat
+import org.junit.jupiter.api.BeforeEach
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.TestInstance
+import org.junit.jupiter.api.assertAll
+
+@TestInstance(TestInstance.Lifecycle.PER_CLASS)
+class LottoStatisticsTest {
+
+    val winningLotto = WinningLotto(Lotto(mutableSetOf(1, 2, 3, 4, 5, 6)))
+    var lottos: MutableList<Lotto> = mutableListOf()
+    lateinit var lottoStatistics: LottoStatistics
+
+    @BeforeEach
+    fun setUp() {
+        lottos.add(Lotto(mutableSetOf(8, 21, 23, 41, 42, 43)))
```

Comment:
```suggestion
        lottos.add(Lotto(setOf(8, 21, 23, 41, 42, 43)))
```

setOf가 아닌 mutableSetOf를 사용한 이유를 알 수 있을까요?

Reply:
`mutableListOf`를 쓰고 당연히 set도 그렇게 써야겠다 생각한 것 같습니다. 🥲